src/Entity/BlockInteractor.php

```diff
@@ -67,18 +67,21 @@ public function getBlocksById(PageInterface $page): array
         return $blocks;
     }
 
-    public function saveBlocksPosition(array $data = [], bool $partial = true): bool
+    public function saveBlocksPosition(array $data = []): bool
     {
         $em = $this->getEntityManager();
         $em->getConnection()->beginTransaction();
 
         try {
             foreach ($data as $block) {
-                if (!isset($block['id'], $block['position'], $block['parent_id'], $block['page_id'])) {
+                if (!isset($block['id'], $block['position']) || !is_numeric($block['position'])) {
```

Comment:
This was broken too, because always wanted 4 parameters. It was due an update of phpstan.